Gemfile

```diff
@@ -27,3 +27,6 @@ group :test do
   gem "poltergeist"
 end
 
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin]
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
Good question and show ruby have having many way to do one things. :smile: 

Any way, change to mingw and mswin platform only now